src/rspack.ts

```diff
@@ -0,0 +1,5 @@
+import type { Options } from './types'
+import unplugin from '.'
+
+// TODO: some upstream lib failed generate invalid dts, remove the any in the future
```

Comment:
Can you send a PR to unplugin as well? Thank you

Reply:
To [unimport](https://github.com/unjs/unimport)?

Reply:
What's the problem with the type? Isn't it related to unplugin?

Reply:
Oh okay, I misunderstood your first question, I'm sorry.

I don't actually know what the problem is. The `any` typing and the comment are present in all the bundler files (see `vite.ts`, `webpack.ts`, `rollup.ts`, …), but I don't know the reason they've been there in the first place.